flexmeasures/ui/utils/view_utils.py

```diff
@@ -86,14 +87,22 @@ def render_flexmeasures_template(html_filename: str, **variables):
         options["downloadFileName"] = f"asset-{asset.id}-{asset.name}"
     variables["chart_options"] = json.dumps(options)
 
-    variables["menu_logo"] = current_app.config.get("FLEXMEASURES_MENU_LOGO_PATH")
+    account: Account | None = (
+        current_user.account if current_user.is_authenticated else None
+    )
+
+    # check if user/consultant has logo_url set
+    if account:
```

Comment:
Now, when we don't have an account, FLEXMEASURES_MENU_LOGO_PATH will not be applied. I believe applying it (like before) should be the default, and potentially setting the logo from account or consultancy account happens as the next step.

Reply:
Sure, I can do that. But could you share a scenario where we might not have an account? The only instance that comes to mind is the login page, since accessing anything else would typically require an account.

Reply:
True, at this moment that is the situation that would not work anymore.

Reply:
Ok, will make an update soon

Reply:
Hi @nhoening this had been worked on, but I don't know why its still not showing here

Reply:
Maybe not in this conversation view, but I see it in the overall view lines 102 and 103 :)

Reply:
Yeah, i also see it when i go to the branch directly, only when I click on the file from here that I cant see it.